build-tests/sparo-outputs/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "sparo-outputs",
+  "description": "Building this project tests sparo command outputs",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "_phase:build": "heft run --only build -- --clean",
+    "_phase:test": "heft run --only test -- --clean",
```

Comment:
Successful output looks like this, which is a bit counterintuitive:
```
 ---- build started ---- 
[build:typescript] Using TypeScript version 5.3.3
[build:lint] Using ESLint version 8.56.0
 ---- build finished (3.839s) ---- 
 ---- test started ---- 
[test:jest] Using Jest version 29.5.0
No tests found, exiting with code 0
[test:jest] 
[test:jest] Run start. 0 test suites
[test:jest] 
[test:jest] Tests finished:
[test:jest]   Successes: 0
[test:jest]   Failures: 0
[test:jest]   Total: 0
 ---- test finished (0.380s) ---- 
-------------------- Finished (4.231s) --------------------
```

The reason is that Jest is not being used; the "tests" actually run via `run-script-plugin` which is in fact invoked for both `heft build` and `heft test`.  A more understandable approach might be to define a custom Heft phase `heft run --only build-test` and not use Jest at all for this project.  This would also allow separate invocation of building (compiling the code) from testing (running the code).